client/components/ui/map.tsx

```diff
@@ -107,7 +107,7 @@ const Map = () => {
     };
 
     if(err) {
-        return <p>Error: "{err}" was thrown while loading Map</p>
+        return <p>Error: &quot;{err}&quot; was thrown while loading Map</p>
```

Comment:
you ran into a merge conflict here--again, i've fixed it for now, but make sure to `git merge main` *before* your PR next time